doc/glossary.rst

```diff
@@ -0,0 +1,62 @@
+:orphan:
+
+.. include:: links.inc
+
+.. _glossary:
+
+==========================
+Glossary
+==========================
+
+.. contents:: Contents
+   :local:
+
+
+MNE-Python core terminology
+===========================
+
+epochs
+    Epoched data; dimensionality: epochs x sensors x times. Typically,
```

Comment:
`n_epochs x n_channels x n_times` to be consistent with `n_events`?

Reply:
I think everywhere we should stick to NumPy shape terminology, e.g.

Epoch data; shape (n_epochs, n_channels, n_time) ...

Reply:
I agree ...